timon/probes.py

```diff
@@ -355,12 +355,15 @@ def parse_result(self, jsonstr):
             return
         self.msg = resp.get('msg') or rslt.get('reason') or ''
         if rslt.get("status") != 200:
+            self.msg += f"cannot get information (http status_code={rslt.get("status")})"
```

Comment:
typical case for the walrus operator. unfortunately still not the right python version.
perhaps 
```
status = rslt.get("status")
if status != 200:
    self.msg += f"cannot get information (http status_code={status})"
```
?

Reply:
Done